admin-manual/installation/execution-limits.rst

```diff
@@ -167,29 +167,23 @@ If you do make changes to the PHP configuration poool and/or the ``php.ini``
 default settings, you should restart PHP-FPM after, and clear your application
 caches and restart Memcached (if you're using it as cache engine).
 
-**In Ubuntu 14.04 with PHP 5.x:**
+**In Ubuntu 16.04 with PHP 7:**
 
 .. code-block:: bash
 
-   sudo service php5-fpm restart
+   sudo systemctl restart php7.0-fpm
    php symfony cc
 
-**In Ubuntu 16.04 with PHP 7:**
+**In Ubuntu 18.04 with PHP 7:**
```

Comment:
I think we should specify 7.0 and 7.2 in these commands. It is technically possible for someone to have 16.04 with PHP 7.2, and vice versa.